tests/features/contact.feature

```diff
@@ -0,0 +1,33 @@
+@api
+Feature: Contact paragraph.
+  As a content editor
+  I need to be able to use Contact paragraphs
+  so that I can reference General and Press Contact entities.
+
+  Scenario: Contact creation.
```

Comment:
Editors can create contacts though paragraphs.

Reply:
Test is missing that both press and general contact types are available.

Reply:
The select doesn't have a label so I used the select name.